hphp/runtime/vm/jit/phys-reg-saver.cpp

```diff
@@ -109,13 +81,9 @@ PhysRegSaver::~PhysRegSaver() {
   }
 }
 
-size_t PhysRegSaver::rspAdjustment() const {
-  return m_adjust;
-}
-
 size_t PhysRegSaver::dwordsPushed() const {
-  assertx((m_adjust % sizeof(int64_t)) == 0);
-  return m_regs.size() + m_adjust / sizeof(int64_t);
+  // Round up the number of regs pushed to an even number
+  return (m_regs.size() + 1) ^ 0x1;
```

Comment:
@dave-estes—This is my fault for not catching this, but this expression is total nonsense:

```
(2 + 1) ^ 1 == 2
(3 + 1) ^ 1 == 5
(4 + 1) ^ 1 == 4
```

and also it doesn't take into account GPR regs vs. XMM regs.  I'll have a fix upstreamed soon.